site/e2e/provisionerGenerated.ts

```diff
@@ -7,1181 +7,1073 @@ export const protobufPackage = "provisioner";
 
 /** LogLevel represents severity of the log. */
 export enum LogLevel {
-	TRACE = 0,
-	DEBUG = 1,
-	INFO = 2,
-	WARN = 3,
-	ERROR = 4,
-	UNRECOGNIZED = -1,
+  TRACE = 0,
```

Comment:
This file is incorrectly indented. It'll be annoying if a developer is making local changes here and the whole file is reformatted.